src/EventListener/ExceptionListener.php

```diff
@@ -37,4 +55,35 @@ public function onKernelException(GetResponseForExceptionEvent $event)
 
         parent::onKernelException($event);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function logException(\Exception $exception, $message)
+    {
+        if (null === $this->logger) {
+            parent::logException($exception, $message);
+
+            return;
+        }
+
+        $exceptionClass = get_class($exception);
+        $statusCode = null;
+
+        foreach ($this->exceptionToStatus as $class => $status) {
+            if (is_a($exceptionClass, $class, true)) {
```

Comment:
You can pass `$exception` directly, remove the call to `get_class` and remove the last parameter.